src/role-helpers.js

```diff
@@ -134,7 +134,13 @@ function getRoles(container, {hidden = false} = {}) {
       return hidden === false ? isInaccessible(element) === false : true
     })
     .reduce((acc, node) => {
-      const roles = getImplicitAriaRoles(node)
+      let roles = []
+      // TODO: This violates html-aria which does not allow any role on every element
```

Comment:
I'd love to deal with this TODO before merging if it's not a huge effort.

Reply:
Oh, just noticed what you said about this. Maybe rather than `TODO` this should link to some explanation for why we're doing this.

Reply:
It is a TODO though. The way we currently handle it is wrong. But this can be fixed in a separate PR so that we're able to revert if multiple roles are required still (also better for changelog).